src/Queue/QueueHandler.php

```diff
@@ -60,78 +66,78 @@ public function __construct(
      */
     public function handleSqs(SqsEvent $event, Context $context): void
```

Comment:
This PR add storing the job attempt in the payload, but it's not adding anything like the [`VaporWorkCommand`](https://github.com/laravel/vapor-core/blob/45efd886ba196e4acb7a218e78e4eb8183358a48/src/Console/Commands/VaporWorkCommand.php).

What are your thoughts on adding this as an optional mechanism instead of SQS event trigger the Lambda method for each new message?

Reply:
So Vapor itself executes the `vapor:work` artisan command on every invocation of the queue function. I skip that, because we already have an entry point (`php/runtime.php`). I didn't see the added value of it.

What would be the added benefit of running `vapor:work` and not have SQS tigger your function?

Reply:
I'm not sure, I assume the Lambda instance triggered by the SQS trigger will remain active and wait for more events for a while, right?

Reply:
That also works in this way :)